src/test/resources/uk/gov/justice/hmpps/casenotes/controllers/A1234AA-casenote.json

```diff
@@ -6,7 +6,7 @@
       "typeDescription": "POM Notes",
       "subType": "GEN",
       "subTypeDescription": "General POM Note",
-      "authorUserId": "SECURE_CASENOTE_USER_ID",
+      "authorUserId": "1111",
```

Comment:
These tests now pick up the user id from the mocked UserDetails response which is `1111`